### PR TITLE
[TRANSLATION] Add/update Hungarian translation for shell32, ieframe, browseui

### PR DESCRIPTION
**Pull request by 7OMI**
Add/update Hungarian translation for shell32, ieframe, browseui

### CODE REVIEW

File: dll/win32/browseui/lang/hu-HU.rc [hunk 1]
```diff
@@ -0,0 +1,355 @@
+/*
+ * Copyright 2009 Andrew Hill
```
Comment (binarymaster) [inline dll/win32/browseui/lang/hu-HU.rc]: I think it's not necessary to copy this license header from en-US translation file. You can add your own, as it was done here:
- https://git.reactos.org/?p=reactos.git;a=blob;f=dll/win32/browseui/lang/pl-PL.rc;hb=c082d08cf34cf09e003009dc5862fc7e0a6c38c1
- https://git.reactos.org/?p=reactos.git;a=blob;f=dll/win32/browseui/lang/es-ES.rc;hb=c082d08cf34cf09e003009dc5862fc7e0a6c38c1

File: dll/win32/browseui/lang/hu-HU.rc [hunk 2]
```diff
@@ -0,0 +1,355 @@
...
+    LTEXT "", IDC_TEXT_LINE + 2, 7, 75, 190, 10, SS_LEFT|SS_NOPREFIX
+    CONTROL "", IDC_PROGRESS_BAR, PROGRESS_CLASSA, WS_BORDER, 7, 65, 190, 8
+    PUSHBUTTON "Cancel", IDCANCEL, 205, 65, 50, 15, WS_GROUP | WS_TABSTOP | WS_VISIBLE
+END
+
+IDD_CUSTOMIZETOOLBAREX DIALOGEX 0, 0, 357, 33
+STYLE DS_SHELLFONT | DS_CONTROL | WS_CHILD | WS_VISIBLE | WS_CAPTION
+FONT 8, "MS Shell Dlg", 0, 0, 0x1
+BEGIN
+    LTEXT "Te&xt options:", -1, 4, 2, 48, 15
```
Comment (binarymaster) [inline dll/win32/browseui/lang/hu-HU.rc]: Some strings are not translated. Is this work in progress?

**Comment by binarymaster** [comment]
Cc recent Hungarian translators @blueghost-tbs, @Tudi20, @xTibor for review.

**Comment by blueghost-tbs** [comment]
Yes, it seems like a work in progress, but definitely in a good direction.

Some thoughts on the PR:
- I don't know if translating "Program Files" and "Application Data" makes any compatibility problems, probably not.
- Back in 2005 the original hungarian translators decided to use "tegezés" instead of "magázás". It's like using "du" instead of using "Sie" in German. I remember there was even a wiki page for that, though it's disappeared since than. I think this decision was made to make ReactOS friendlier. So in my work I always tried to stick with that decision. You should too. ;)

Other than that, it's a proper WIP translation, thanks!

**Comment by HBelusca** [comment]
Personally I prefer making ReactOS more professional, with correct grammar and orthograph, and respectful of the user, so --> "Sie" (German) or the equivalent in Hungarian.
Pseudo-familiarity with the user needs to be proscribed (when I see such software I am really rebuked to use them).

**Comment by blueghost-tbs** [comment]
@HBelusca : if this is what main developers wish, then so be it.

**Comment by SergeGautherie** [comment]
Missing author real name!?

**Pull request merged**